src/screens/Topics/index.js

```diff
@@ -187,39 +190,40 @@ const Topics = () => {
   );
 };
 const styles = StyleSheet.create({
-  scrollViewStyle: {marginBottom: 100},
+  scrollViewStyle: (bottom) => ({
+    marginBottom: dimen.normalizeDimen(100) - bottom
+  }),
   header: {
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center'
   },
   container: {
     flex: 1,
-    padding: Platform.OS === 'ios' ? 22 : 0,
+    padding: Platform.OS === 'ios' ? dimen.normalizeDimen(22) : 0,
     backgroundColor: colors.white
   },
   containerProgress: {
-    marginTop: 20,
-    marginBottom: 24,
-    paddingHorizontal: 20
+    marginTop: dimen.normalizeDimen(20),
+    marginBottom: dimen.normalizeDimen(24),
+    paddingHorizontal: dimen.normalizeDimen(20)
   },
   textPickYourTopic: {
     fontFamily: 'Inter-Bold',
     fontStyle: 'normal',
     fontWeight: '600',
-    fontSize: 36,
-    lineHeight: 43.57,
+    fontSize: normalizeFontSize(36),
+    lineHeight: normalizeFontSize(43.57),
     color: '#11243D',
-    marginHorizontal: 20
+    marginHorizontal: dimen.normalizeDimen(20)
   },
   footer: {
     position: 'absolute',
     bottom: 0,
-    height: 112,
+    height: dimen.normalizeDimen(112),
     width,
-    paddingLeft: 20,
-    paddingRight: 20,
-    paddingBottom: 20,
+    paddingHorizontal: dimen.normalizeDimen(20),
+    paddingBottom: dimen.normalizeDimen(20),
     backgroundColor: colors.white,
     shadowColor: colors.black,
     shadowOffset: {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [193-271]

The changes in this hunk are all about replacing hard-coded values with normalized dimensions using the `dimen.normalizeDimen` function and adjusting various style and layout properties such as margins, padding, widths, heights, and font sizes. These changes will improve the responsiveness and visual consistency of the components across different screen sizes and resolutions.
```diff
+   scrollViewStyle: (bottom) => ({
+     marginBottom: dimen.normalizeDimen(100) - bottom
+   }),
...
+     fontSize: normalizeFontSize(14),
+     lineHeight: normalizeFontSize(24),
...
+     marginBottom: dimen.normalizeDimen(32)
...
+     fontSize: normalizeFontSize(18),
+     lineHeight: normalizeFontSize(22),
...
+     marginBottom: dimen.normalizeDimen(8),
+     paddingRight: dimen.normalizeDimen(8)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->